nemo/src/execution/execution_engine.rs

```diff
@@ -45,6 +45,8 @@ pub struct ExecutionEngine<RuleSelectionStrategy> {
 
     rule_strategy: RuleSelectionStrategy,
 
+    #[allow(dead_code)]
```

Comment:
Is this still `dead_code`? Looks like it's used below, so the attribute could go?